recipes/kafka.rb

```diff
@@ -34,7 +34,7 @@
   elsif node['cdap']['cdap_site'].key?('kafka.log.dir')
     node['cdap']['cdap_site']['kafka.log.dir']
   else
-    '/tmp/kafka-logs'
+    '/data/cdap/kafka-logs'
   end
 
 node.default['cdap']['cdap_site']['kafka.server.log.dirs'] = kafka_log_dirs
```

Comment:
suggest changing this to ``[ *kafka_log_dirs ].join(',')``.  With the above defaults removed, it is no longer clear whether ``kafka.server.log.dirs`` should be a string or array.  This will handle both.  (and assuming its not used anywhere else)